sda/cmd/api/api_test.go

```diff
@@ -710,6 +710,37 @@ func testEndpoint(c *gin.Context) {
 	c.JSON(200, gin.H{"ok": true})
 }
 
+func (s *TestSuite) TestLogLevelDebug() {
+	// Capture log output from gin in buf
+	var buf bytes.Buffer
+	log.SetOutput(&buf)
+	// Ensure we log to stdout after this test is completed
+	defer log.SetOutput(os.Stdout)
+
+	log.SetLevel(log.DebugLevel)
+	srv := setup(Conf)
+	srv.Addr = ":8080" // TODO: What is the proper way to configure this from api_test.go?
+	go func() {
+		if err := srv.ListenAndServe(); err != nil {
+			log.Fatal(err)
+		}
+	}()
+
+	// Allow some time for startup
+	time.Sleep(500 * time.Millisecond)
+
+	// Send a request towards previously started *http.Server
+	_, err := http.Get("http://localhost:8080/files")
+	assert.NoError(s.T(), err)
+
+	// Allow logs to flush
+	time.Sleep(200 * time.Millisecond)
+
+	// Inspect output
+	logOutput := buf.String()
+	fmt.Println("log output: ", logOutput) // TODO: How do we consider a pass? Regexp and match the output?
```

Comment:
Correct, for `debug` the log lines should start with `[Gin] <DATE>` unless the `/ready` path is called.
For any other log level there should be no `[Gin] <DATE>` in the log lines.

So for a full coverage you need to test two endpoints in debug mode and one in `info` and ensure that the log lines look as we expect them to.